package-lock.json

```diff
@@ -45,6 +46,7 @@
         "localforage": "^1.10.0",
         "lodash": "4.17.21",
         "lottie-react-native": "^5.1.5",
+        "magic-bytes.js": "^1.0.14",
```

Comment:
Why would we need this?
Your proposal doesn't give us any hint of using a third-party library

Reply:
Also, it's better to include why would we need these new dependencies in the PR description.

Reply:
Instead of using custom mappings for file signatures in our code, its better to use something that has been well tested and is maintained. The `magic-bytes` library has the mappings in it. The `Buffer` library is used to convert file content  (which is a string) to a `Buffer` such that the data can be compared with the mappings.

Reply:
@Santhosh-Sellavel Updated the description.

Reply:
Cool thanks for updating, let's wait for @PauloGasparSv's thoughts too!